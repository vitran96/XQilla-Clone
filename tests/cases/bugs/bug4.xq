copy $a := document { "   ", processing-instruction hello {""}, <foo/>, comment { "Ok" } }
modify ()
return (
  count($a/self::document-node()),
  $a treat as document-node()
)
