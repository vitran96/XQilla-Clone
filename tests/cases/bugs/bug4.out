1   <?hello ?><foo/><!--Ok-->
